{"name": "not authenticated user -- @1.12 cases", "status": "passed", "steps": [{"name": "Given alien makes get request: /reconciliation/services_compensations/", "status": "passed", "start": 1610716072194, "stop": 1610716072222}, {"name": "Then check response of request: Authentication credentials were not provided. is True", "status": "passed", "start": 1610716072222, "stop": 1610716072222}], "parameters": [{"name": "url", "value": "/reconciliation/services_compensations/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "True"}], "start": 1610716072193, "stop": 1610716072222, "uuid": "92631a2b-6f6f-40e1-9ca1-1c3a2223e3eb", "historyId": "7d966bca3d43e7928063c8ea050cd1d9", "fullName": "features/II-Reconciliation/III-reconciliation_access.feature:not authenticated user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}